### ISSUE TITLE
Display of notes in epub

### ISSUE THREAD

**Issue by tonyahowe** [LiteratureInContext/LiC-app#146]
Notes are being displayed in-line in epub. I don't know how epubs work and what the best way is to access notes, but at the bottom or the side or by click and return at the end? Do epubs allow popups?
<img width="1146" alt="Screen Shot 2022-10-05 at 9 57 10 AM" src="https://user-images.githubusercontent.com/1153464/194078986-d9f00a14-42f4-4dc3-a2e4-6b1e671a31f8.png">

**Comment by wsalesky** [comment]
@tonyahowe I have a fix for the footnote issue, but the EPUB layout needs a lot of work in general. How much time do you want put into it? I would estimate up to 5 hours to get it to look and work correctly.

**Comment by tonyahowe** [comment]
I think 5 hours sounds like a plan! Yeah, let’s do that. There are a few tweaks to the xslt - html that shouldn’t take too long, too; I’ll make an issue for that.

**Comment by wsalesky** [comment]
@tonyahowe Has this issue been resolved? I'm doing some issue cleanup.

**Comment by tonyahowe** [comment]
I will look at it today!

**Comment by tonyahowe** [comment]
I don't think so-- here's a SS of an epub generated from Doyle's _Sign of Four_; the notes are still in line, and the images in notes are not included (though indications of their presence are visible). 
![Screenshot 2024-02-13 at 9 51 18 AM](https://github.com/LiteratureInContext/LiC-app/assets/1153464/71f4e56d-5a93-4c4b-92c0-6a3f2c9c6469)

I downloaded this to my Mac laptop and am looking at it in Books.

**Comment by wsalesky** [comment]
@tonyahowe This should be fixed now.

**Comment by wsalesky** [comment]
@tonyahowe Checking in on this one also.
